server/initializers/constants.ts

```diff
@@ -441,12 +441,12 @@ let CONSTRAINTS_FIELDS = {
 
 const RATES_LIMIT = {
   LOGIN: {
-    WINDOW_MS: 5 * 60 * 1000, // 5 minutes
-    MAX: 15 // 15 attempts
+    WINDOW_MS: config.get<number>('webserver.rates_limit.login.window_ms'),
```

Comment:
use `parseDuration` so you can set `5 minutes` or `1 hour` in configuration

Reply:
Should I then rename the config property in `window`, given the value will not be expressed in `ms` anymore?

Reply:
yep